src/validators/stops.rs

```diff
@@ -133,7 +131,7 @@ fn test_missing() {
         .collect();
 
     assert_eq!(1, missing_coord_issue.len());
-    assert_eq!("AMV", missing_coord_issue[0].object_id);
+    assert_eq!("null_island", missing_coord_issue[0].object_id);
```

Comment:
❤️ love null island !

I would probably add a test explicitly allowing DIJON/AMV and verifying that they have one zero and not two in their coordinates.

If that is a bit too complicated to write, just a comment on ligne above, since this is also indirectly tested by the fact that `missing_cord_issue.len() == 1` at the moment (but this is less explicit, and would not protect us from a regression if someone else removes `AMV/DIJON` from the test set in the future, unknowingly.